config/account.go

```diff
@@ -0,0 +1,129 @@
+package config
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+type AccountsFile struct {
+    ActiveToken string
+    Accounts []*Account
+}
+
+// Account has a name and a token. The name is just for the users recognition.
+// The token is the actual token used to authenticate against the discord API.
+type Account struct {
+	Name  string
+	Token string
+}
+
+var LoadedAccountsFile *AccountsFile
+var cachedAccountsFile string
+
+// SetAccountsFile sets the accounts file path cache to the
+// entered value
+func SetAccountsFile(accountsFilePath string) error {
```

Comment:
This code was copied and we should avoid duplication. If there's a bug in this code, it most likely won't be fixed for config. The same goes for all other code that was copied.